### PR TITLE
EE-1249 fixing PCP font sizes

### PR DESCRIPTION
**Pull request by cameronpettit**

### CODE REVIEW

File: src/app/comments/comments.component.html [hunk 1]
```diff
@@ -9,12 +9,13 @@
...
+            <h2>
               {{commentPeriodHeader}}
-            </h3>
-            <h3>
-              {{commentPeriod.dateStarted | date:'mediumDate'}} - {{commentPeriod.longEndDate.format('MMMM DD @ hh:mm A z')}}
-            </h3>
+            </h2>
+            <h2>
+              {{commentPeriod.dateStarted | date:'mediumDate'}} - {{commentPeriod.longEndDate.format('MMMM DD @ hh:mm A
+              z')}}
```
Comment (marklise) [inline src/app/comments/comments.component.html:17]: Single line here plz

File: src/app/comments/comments.component.html [hunk 2]
```diff
@@ -27,7 +28,8 @@ <h3>
               </p>
             </div>
             <div *ngIf="commentPeriod.informationLabel === null">
-              <div id="instructions" [innerHTML]="commentPeriod.instructions  | safeHtml"></div>
+              <div id="instructions" [innerHTML]="commentPeriod.instructions  | safeHtml">
+              </div>
```
Comment (marklise) [inline src/app/comments/comments.component.html:32]: Same

**Review by marklise — APPROVED** [review]
/w nits

**Review by BcGovNeal — APPROVED** [review]

File: src/app/comments/comments.component.html (hunk 1 above)
Comment (cameronpettit) [reply src/app/comments/comments.component.html]: Code formatted

**Pull request merged**